### ISSUE TITLE
Game Specific button remap saves - Feature Request

### ISSUE THREAD

**Issue by Jimiga** [MiSTer-devel/Minimig-AGA_MiSTer#120]
Is it a possibility that we might be able to save button remapping per-game in minimig? 

I really like the current feature of being able to assign 'up' to a button for jump, likewise space bar for bombs etc in some games, but this currently resets after each session. 

I'd be interested to hear!

**Comment by sorgelig** [comment]
It's made to be a temporary feature, per game. Gore doesn't know which game is running and thus cannot save it.

**Comment by Jimiga** [comment]
Thanks for the reply, good to know how this works.

**Comment by mgolfieri** [comment]
Makes sense. Then we could expand the launch configuration to also define the floppy disks (and thus the game within it) and the remapping done?
